packages/dd-trace/test/dd-trace.spec.js

```diff
@@ -35,7 +35,6 @@ describe('dd-trace', () => {
       expect(payload[0][0].start).to.be.instanceof(Uint64BE)
       expect(payload[0][0].duration).to.be.instanceof(Uint64BE)
       expect(payload[0][0].metrics).to.have.property(SAMPLING_PRIORITY_KEY)
-      expect(payload[0][0].meta).to.have.property(DECISION_MAKER_KEY, '-0')
```

Comment:
This check is test execution order dependant, is that property that we were trying to test here?

Reply:
It shouldn't be test order dependent, if it is it would be a bug 🤔 the idea was that every trace should have this property.

Reply:
It has the property, but with other value, with `-1` instead of `-0`